### PR TITLE
comment page on BO was created

### PR DESCRIPTION
**Pull request by d-victor**
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Dehtiarov/BlogVideh/Model/Comment.php [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+ * @package Dehtiarov\BlogVideh\Model\Post
+ *
+ */
+class Post extends AbstractModel implements CommentInterface, IdentityInterface
+{
+    /**#@+
+     * Post's Statuses
+     */
+    const STATUS_ENABLED = 1;
+    const STATUS_DISABLED = 0;
```
Comment (RomaChernii) [inline app/code/Dehtiarov/BlogVideh/Model/Comment.php]: в нас буде 3 статуса : New, In progress, Closed

File: app/code/Dehtiarov/BlogVideh/Model/ResourceModel/Comment/Collection.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    /**
+     * Define resource model
+     *
+     * @return void
+     */
+    public function _construct()
+    {
+        $this->_init('Dehtiarov\BlogVideh\Model\Comment', 'Dehtiarov\BlogVideh\Model\ResourceModel\Comment');
+    }
+}
```
Comment (RomaChernii) [inline app/code/Dehtiarov/BlogVideh/Model/ResourceModel/Comment/Collection.php]: пуста строка в кінці

File: app/code/Dehtiarov/BlogVideh/Model/ResourceModel/Comment/Grid/Collection.php [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+     *
+     * @return $this
+     *
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function setItems(array $items = null)
+    {
+        return $this;
+    }
+}
```
Comment (RomaChernii) [inline app/code/Dehtiarov/BlogVideh/Model/ResourceModel/Comment/Grid/Collection.php]: пуста строка

**Comment by d-victor** [comment]
исправил и докинул грид комментов

**Pull request merged**